app/models/course/registration.rb

```diff
@@ -0,0 +1,29 @@
+class Course::Registration < ActiveRecord::Base
+  stampable
+
+  scope :staff, -> { where(role: roles.values_at(:teaching_assistant, :manager, :owner)) }
```

Comment:
Did you mean when Course::Registration.staff, it returns something like [1, 2, 3] where the 1, 2 and 3 are role numbers?